study-builder/tenants/rarex/login.html

```diff
@@ -0,0 +1,512 @@
+                "style",
+                "margin-top:0;font-size: small;text-align: left;"
+              );
+
+              if (showSignUpHelp) {
+                var registerMessage = document.createElement("p");
+                registerMessage.setAttribute("style", "margin-bottom:0;");
+                registerMessage.innerHTML =
+                  "If you haven't registered yet, please <a class='auth0-lock-alternative-link' href='" +
+                  registerUrl +
+                  "'>do so here</a>.";
+                registerMessageWrapper.appendChild(registerMessage);
+
+                if (lockForm) {
+                  lockForm[0].appendChild(registerMessageWrapper);
+                }
+              }
+            }
+
+            var helpWrapper = document.createElement("div");
+            helpWrapper.setAttribute("id", "ddp-help");
+            helpWrapper.setAttribute(
+              "style",
+              "display:flex;flex-direction:row;align-items:center;margin-top:5px;flex-wrap:none;"
+            );
+            var help =
+              '<p style="text-align:left;margin-bottom:0;">For help, please contact ' +
+              studyHelpEmail +
+              ".</p>";
+            helpWrapper.innerHTML = help;
+            if (lockForm) {
+              lockForm[0].appendChild(helpWrapper);
+            }
+
+            if (mode === "signup" || options.allowSignUp) {
+              console.log("SIGN UP MODE");
+              console.log("appending sign in with existing account");
+              var loginLink = document.createElement("a");
+              loginLink.setAttribute("class", "auth0-lock-alternative-link");
+              loginLink.setAttribute("href", loginUrl);
+              loginLink.innerText = "Sign in with existing account.";
+              var switchToLogin = document.createElement("p");
+              switchToLogin.setAttribute("class", "auth0-lock-alternative");
+              switchToLogin.setAttribute(
+                "style",
+                "text-align:left;margin-top:5px;"
+              );
+              switchToLogin.innerText = "Already registered? ";
+              switchToLogin.appendChild(loginLink);
+              helpWrapper.parentElement.appendChild(switchToLogin);
+
+              var signUpWidget = document.getElementsByClassName(
+                "auth0-lock-widget"
+              )[0];
+              signUpWidget.style.width = "500px";
+              var emailField = document.getElementsByClassName(
+                "auth0-lock-input-email"
+              )[0];
+              var firstNameField = document.getElementsByClassName(
+                "auth0-lock-input-first_name"
+              )[0];
+              var lastNameField = document.getElementsByClassName(
+                "auth0-lock-input-last_name"
+              )[0];
+              var fieldsDiv = document.getElementsByClassName(
+                "auth0-lock-form"
+              )[0].childNodes[0].childNodes[2];
```

Comment:
There's a potential issue here. This code assumes DOM structure when google-oauth connection is enabled. I'm not sure if that connection should be enabled or not.